README.md

```diff
@@ -124,7 +139,64 @@ mvn test -Dtest=SanityTests -X
         
   </details>
 
-<br/><br/>
+</br>
+
+> If you weren't successful at setting up you local env, then use the [Gitpod approach](#gitpod-setup)👇
+
+
+### Gitpod setup
+
+Open forked repo in Gitpod.io
+
+1. Sign up for a free [Github account](https://github.com/)
+2. Fork this repository
+    * Make sure you are logged into Github
+    * Click the fork in the upper right of Github
+    * Select your username as the location to fork the repo
+3. In the browser address bar, prepend the Github url (`https://github.com/<username>/automation-best-practices-java`) with `https://gitpod.io/#`
+    * The resulting url should look as follows:
+   
+      > gitpod.io/#https://github.com/{USERNAME}/automation-best-practices-java
+   
+4. Once the Gitpod.io URL is loaded, you will need to sign in with the Github account you created earlier 
+5. Once the development environment is loaded and you see 'Ready to test!', run the following commands to set your `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY`:
```

Comment:
Also missing Visual API key creating the following failure. I reference the Screener stuff here https://github.com/saucelabs-training/automation-best-practices-java#sign-up-for-accounts .
![Screen Shot 2021-11-09 at 2 33 20 PM](https://user-images.githubusercontent.com/2136245/140992164-a97789f9-9096-4a7d-80c5-b3f323b54fb7.png)

Reply:
@nadvolod added information about how to get your Screener API key